### PR TITLE
DOC: add example to reorder levels

### LINKED ISSUE #42124: DOC add example to reorder levels
https://pandas.pydata.org/pandas-docs/dev/reference/api/pandas.DataFrame.reorder_levels.html?highlight=reorder_levels#pandas.DataFrame.reorder_levels

No examples! Can we add at least one?

### PR DESCRIPTION
**Pull request by prabha-git**
- [ x] closes #42124
- [ ] tests added / passed
- [x ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#code-standards) for how to run them
- [ x] whatsnew entry - Included example for reorder_levels

### CODE REVIEW

File: pandas/core/frame.py [hunk 1]
```diff
@@ -6835,6 +6835,52 @@ def reorder_levels(self, order: Sequence[Axis], axis: Axis = 0) -> DataFrame:
...
+            2	    a	0.748423
+                    b	1.468277
+                    c	0.077554
+
+
+        In the following example, we will reorder the levels by referencing index positions.
+        
+        >>> df.reorder_levels([1,0])
+		                col
+        index_2	index_1
```
Comment (jreback) [inline pandas/core/frame.py:6864]: are these actually how this is formatted? i believe we align

Comment (prabha-git) [reply pandas/core/frame.py]: Updated the formatting @jreback , Can you check now.

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
Can you fixup the CI failure please?

```
=================================== FAILURES ===================================
_____________ [doctest] pandas.core.frame.DataFrame.reorder_levels _____________
6840         --------
6841         >>> df = pd.DataFrame(
6842         ...     {'col' : pd.Series(np.random.randn(6),
6843         ...                 index = [
6844         ...                             [1,1,1,2,2,2],
6845         ...                             ['a','b','c','a','b','c']
6846         ...                           ]
6847         ...                         )})
6848         >>> df.index.names = ['index_1','index_2']
6849         >>> df
Differences (unified diff with -expected +actual):
    @@ -1,8 +1,8 @@
    -                        col
    -index_1     index_2
    -    1       a           1.763918
    -            b           -0.195622
    -            c           -1.675671
    -    2       a           0.748423
    -            b           1.468277
    -            c           0.077554
    +                      col
    +index_1 index_2          
    +1       a        1.199896
    +        b        0.643956
    +        c        0.098624
    +2       a        1.057967
    +        b       -0.265410
    +        c       -0.116204

/home/runner/work/pandas/pandas/pandas/core/frame.py:6849: DocTestFailure
```

**Comment by MarcoGorelli** [comment]
closing for now as stale

**Pull request closed without merging**